data_generation/download_tasks.py

```diff
@@ -0,0 +1,66 @@
+"""This file downloads the requested tasks"""
+import os
+
+from absl import app
+from absl import flags
+from absl import logging
+
+import inductiva
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string("path_to_sim_ids", None,
+                    "The path to the simulation ids txt files.")
+
+flags.DEFINE_string(
+    "download_dir", None,
+    "Where to download the data. If None, the data is not downloaded.")
+
+
+def main(_):
+    inductiva.api_key = os.environ["INDUCTIVA_API_KEY"]
+
+    # Read the simulation ids
+    with open(FLAGS.path_to_sim_ids, "r", encoding="utf-8") as f:
```

Comment:
I feel that this should be a feature we should have... Given a set of task_ids, fetch all task objects in one go. But given the simplicity I am not sure... Wdyt as a user?

Reply:
I feel it is simple enough to fetch them one by one

Reply:
Alright!